xml/Accessibility/AnnoScope.xml

```diff
@@ -2,6 +2,8 @@
   <TypeSignature Language="C#" Value="public enum AnnoScope" />
   <TypeSignature Language="ILAsm" Value=".class public auto ansi sealed AnnoScope extends System.Enum" />
   <TypeSignature Language="DocId" Value="T:Accessibility.AnnoScope" />
+  <TypeSignature Language="VB.NET" Value="Public Enum AnnoScope" />
+  <TypeSignature Language="F#" Value="type AnnoScope = " />
```

Comment:
MSDN: type AnnoScope

Reply:
In this case, the ` = ` is different, but still correct.  F# enums are defined like this:

``fsharp
type MyEnum =
    | Val1 = 1
    | Val2 = 2
    | Val3 = 3
```

MSDN showed these without the ` = `.  I'm not sure which is better or worse, so this is probably okay.